content/departments/engineering/teams/code-exploration/index.md

```diff
@@ -13,11 +18,65 @@
 ## Contact
 
 - #code-exploration channel in Slack
-<!-- - @code-exploration-support to ping the engineer on weekly support rotation. -->
+- #integrations channel for integration-specific discussion
+<!-- - TODO: @code-exploration-support to ping the engineer on weekly support rotation. -->
 - [<span class="badge" style="background: #c2255c; color: white; font-weight: bold">team/code-exploration</span>](https://github.com/sourcegraph/sourcegraph/issues/new?labels=team/code-exploration) label on GitHub
 - @sourcegraph/code-exploration team on GitHub
 - code-exploration@sourcegraph.com Google group for mentioning in Google Docs and calendar invites
 - [Code Exploration Events](https://calendar.google.com/calendar/u/0?cid=Y181MmU3ZmZmY2UzMWEyZjU1ZDMyOTExYjVkYmMxOTI0OTk1NzI1OWRmYjU3NWMzMmVhYTg0YzNkMTBmNWRiMWZlQGdyb3VwLmNhbGVuZGFyLmdvb2dsZS5jb20) calendar
-  - Teammates should add this calendar (calendar ID `c_52e7fffce31a2f55d32911b5dbc19249957259dfb575c32eaa84c3d10f5db1fe@group.calendar.google.com`) as an additional calendar in the [Roots PTO Slack app](../../../../benefits-pay-perks/benefits-perks/time-off/submitting-time-off.md) so that their PTO is visible to their team on the team calendar (and disable adding their manager directly to the calendar invite).
 - #code-exploration-internal for team (non-product) related discussion
-<!-- - #code-exploration-eng for higher bandwith communication typically amongst the engineering team / PM. This channel is separated out of #code-exploration since this discussion can get quite noisy for the casual observer of #code-exploration. -->
+
+## Ownership
+
+The Code Exploration team owns the core Sourcegraph code exploration UI, which includes the following vertical areas:
+
+- Repository routes
+  - Revision selector
+  - Breadcrumbs
+  - Blob view
+    - Hover popup
```

Comment:
Minor, but I believe historically we have referred to this as the "hover overlay" or "code intel overlay" on FPT (the React component itself is called `HoverOverlay`). Happy to use this to refer to it, as long as everyone is on the same page so we know what our common language to use is. 🙂

Reply:
I don't care much either way. I hear most people refer to it as "hover tooltip", which is always a bit confusing because it's not really a tip for a _tool_ (like a button), and it doesn't behave like an ARIA tooltip role. I guess I wrote popup from the language used in `aria-haspopup="dialog"`.

Reply:
I also agree using "tooltip" is the more confusing option (and my least favorite). So long as we consistently know what we're calling it (especially if we can use that to help other folks/teams know what to call it), I'm happy. 😄